docs/platforms/react-native/sourcemaps/uploading/expo.mdx

```diff
@@ -17,7 +17,7 @@ Sentry's React Native SDK works out of the box with Expo applications. To see re
 
 ## Automatic Upload for Release
 
-When the Sentry Expo Plugin `@sentry/react-native/expo` and the Sentry Metro Pugin are added to your application configuration, source maps for the native builds are uploaded automatically during EAS Builds and when building the native application release locally using `npx expo prebuild`.
+When the Sentry Expo Plugin `@sentry/react-native/expo` and the Sentry Metro Pugin are added to your application configuration, source maps for the native builds are uploaded automatically during EAS Builds and when building the native application release locally using `npx expo prebuild`. Note that uploading source maps for debug builds is disabled, as symbolication is handled by the Metro dev server.
```

Comment:
```suggestion
When the Sentry Expo Plugin `@sentry/react-native/expo` and the Sentry Metro Plugin are added to your application configuration, source maps for the native builds are uploaded automatically during EAS Builds and when building the native application release locally using `npx expo prebuild`. Because symbolication is already handled by the Metro dev server, automatic uploading of source maps for debug builds is disabled.
```